content/docs/command-reference/repro.md

```diff
@@ -127,7 +127,10 @@ and only execute the final stage.
 - `-P`, `--all-pipelines` - reproduce all pipelines, for all the stage files
   present in `DVC` repository.
 
-- `--ignore-build-cache` - in cases like `... -> A (changed) -> B -> C` it will
+- `--no-run-cache` - Run changed stage even if it has been already ran with the
```

Comment:
Very sloppy explanation copied from the --help :slightly_frowning_face: https://github.com/iterative/dvc/pull/3788#discussion_r424051444

Reply:
Run -> run

good for now, I think ... we'll def need to explain the whole concept of the run cache here on in the UG - it has a lot of moving pieces. It'll be easier to explain this option better after that

please, create a ticket with some summary of the run cache - what commands does it affect, etc?

Reply:
Please see https://github.com/iterative/dvc/pull/3788#pullrequestreview-411108828

Reply:
@shcheklein Created https://github.com/iterative/dvc.org/issues/1289